### DISCUSSION TITLE
Copilot Pycharm plugin crashs

### DISCUSSION THREAD

**Discussion by Butanium** [community/community#15444] [Copilot Conversations] [answered]
Everytime I open my file I get this message and the plugin stop working and I have to restart Pycharm (disabling - enabling the completion doesn't work)

```
Error parsing JSON-RPC message: {"jsonrpc":"2.0","id":3,"result":{"completions":[{"uuid":"dfcf9ee2-20d5-4cf9-a1ed-
34c6d6461554","text":"        # Else move randomly around your (franÃ§ais : poste d'observation) (english : observation post)","range":
{"start":{"line":128,"character":0},"end":{"line":128,"character":87}},"displayText":"observation post)","position":{"line":128,"character":85}}]

com.google.gson.JsonSyntaxException: java.io.EOFException: End of input at line 1 column 363 path $.result.completions
	at com.google.gson.Gson.fromJson(Gson.java:942)
```
(full trace [here](https://pastebin.com/JfurJ7Hx))
Here is the line that causes the problem : 
```py
   else:
-->     # Else move randomly around your (français : poste d'observation) (english : 
        normalized_dx, normalized_dy = dx / sqrt(dx ** 2 + dy ** 2), dy / sqrt(dx ** 2 + dy ** 2)
        random_theta = random() * pi - pi / 2
```
Even opening the tab on the right (no matter the code) makes it crash :
![image](https://user-images.githubusercontent.com/55806347/164912528-eab8c22b-19c5-459b-aea6-b3e17c1a6659.png)

What can I do about this ? I see no repo for the plugin where I could open an issue

**Comment by jansorg** [comment, marked as answer]
Thanks! We're looking into it...
In the mean time, you could downgrade to 1.1.16, e.g. by downloading the file from https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/167546 and drag&drop it onto your IDE window to install, then restart.

**Comment by jansorg** [comment]
@Butanium Which OS are you using?

**Reply by Butanium** [reply]
Windows 10, with the latest version of Pycharm 2022.1

**Comment by jansorg** [comment]
The latest release, 1.1.19, has a fix for this problem. Thank you for your patience!
